src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Sources/QueryFileSystemInfo.cs

```diff
@@ -12,41 +12,22 @@ namespace Microsoft.Plugin.Folder.Sources
 {
     public class QueryFileSystemInfo : DirectoryWrapper,  IQueryFileSystemInfo
     {
-        public IEnumerable<DisplayFileInfo> MatchFileSystemInfo(string search, string incompleteName, SearchOption searchOption)
+        public IEnumerable<DisplayFileInfo> MatchFileSystemInfo(string search, string incompleteName, bool isRecursive)
```

Comment:
[Discussion Point] Instead of using a boolean, could we create our own SearchOption enum or class?  Its slightly more extensible than switching on a boolean value.

Reply:
I was thinking about it, but couldn't come up with another value of it. Otherwise I could return the SearchOptions, because that is already the enum with the options. So I'm neutral for your point :)

Reply:
I don't feel strongly about it. The only case I could imagine is if we ever wanted to expose some of the enumeration options to user settings, but I don't see that happening any time soon.